README.md

```diff
@@ -49,19 +49,12 @@ make checks
 make unit-test
 ```
 
-### Running bdd tests
-Instructions for running the bdd tests can be [found here](docs/test/bdd_instructions.md).
+## Table of Contents
```

Comment:
It seems a bit odd to me to see a Table of Contents part way through the page... Maybe this should be moved above the build section? And then the build section could be added to it.